packages/p2p/src/components/orders/order-table/order-table-content.jsx

```diff
@@ -5,7 +5,7 @@ import { reaction } from 'mobx';
 import { isMobile } from '@deriv/shared';
 import { observer } from 'mobx-react-lite';
 import { Localize, localize } from 'Components/i18next';
-import { TableError } from 'Components/table/table-error.jsx';
+import { TableError } from 'Components/table/table-error.tsx';
```

Comment:
The TableError component is still a JavaScript component so until we convert it to a TS component, we can't change the file extension.

Reply:
I was testing to see that everything works fine
I'll convert it to js